k8s/configmap.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flink-config
+  labels:
+    app: flink
+data:
+  flink-conf.yaml: |+
+    parallelism.default: 2
+    jobmanager.memory.process.size: 4048m
+    taskmanager.memory.process.size: 4048m
+    taskmanager.numberOfTaskSlots: 1
+    taskmanager.state.local.root-dirs: /process-state/local-recovery
+    jobmanager.rpc.address: flink-jobmanager
+    taskmanager.rpc.port: 6122
+    jobmanager.rpc.port: 6123
+    blob.server.port: 6124
+    queryable-state.proxy.ports: 6125
+    restart-strategy: fixed-delay
+    restart-strategy.fixed-delay.attempts: 1
+    state.backend: rocksdb
+    state.backend.rocksdb.localdir: /process-state/rocksdb
+    state.backend.incremental: true
+    state.backend.local-recovery: true
+    state.backend.persistent-volume-support: true
```

Comment:
This is not needed.

Reply:
Not needed at all or not needed for baseline tests?

Reply:
Removed all other than rocksdb localdir